cusub_cortex/state_machine/src/tasks/octagon.py

```diff
@@ -138,15 +138,11 @@ def visual_servo_method(self, userdata):
         goal.camera = goal.DOWNCAM
         goal.target_pixel_threshold = self.target_pixel_threshold
         goal.target_frame = rospy.get_param("~robotname") +"/description/downcam_frame_optical"
+        goal.area_axis = goal.NO_AXIS
```

Comment:
Yes, or to accidentally delete one, and then be confused for 1 hour as to why the sub can't do octagon.  Not sure why this is added, technically I deleted it and put it back.

Reply:
Yeah so maybe an interface or helper class to create this msg would be worthwhile, to avoid random issues like that down the line. I can open an issue and add that